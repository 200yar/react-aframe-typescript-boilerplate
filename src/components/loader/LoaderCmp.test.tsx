import React from 'react';
import ReactDOM from 'react-dom';
import Loader from './LoaderCmp';
import renderer from 'react-test-renderer';

describe('<Loader/>', () => {

  it('Smoke test: renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Loader/>, div);
  });

  it('Matches snapshot', () => {
    const componentSnapshot = renderer.create(<Loader>Loading</Loader>).toJSON();
    expect(componentSnapshot).toMatchSnapshot();
  });

});