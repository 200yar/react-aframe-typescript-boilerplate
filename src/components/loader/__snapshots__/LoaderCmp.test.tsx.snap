// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> Matches snapshot 1`] = `
<div
  className="fade-in"
  id="loader-element"
  style={
    Object {
      "position": "absolute",
      "textAlign": "center",
      "top": "41%",
      "width": "100%",
      "zIndex": 11,
    }
  }
>
  <div
    style={
      Object {
        "backgroundColor": "white",
        "borderRadius": "4px",
        "boxShadow": "grey 0 0 10px 0",
        "color": "black",
        "display": "block",
        "fontFamily": "verdana, sans-serif",
        "fontSize": "small",
        "margin": "auto",
        "padding": "20px",
        "textAlign": "center",
        "width": "70px",
      }
    }
  >
    <img
      src="img/history3.svg"
      style={
        Object {
          "verticalAlign": "middle",
        }
      }
    />
    <div
      style={
        Object {
          "paddingTop": "5px",
        }
      }
    >
      Loading
    </div>
  </div>
</div>
`;
